client/src/stories/RequestForm.stories.js

```diff
@@ -1,11 +1,5 @@
 import React from "react";
 import { storiesOf } from "@storybook/react";
 import RequestForm from "../components/RequestForm";
-import GlobalStyle from "../GlobalStyle";
 
-storiesOf("Request form", module).add("Request form", () => (
-  <>
-    <GlobalStyle />
-    <RequestForm />
-  </>
-));
+storiesOf("Request form", module).add("Request form", () => <RequestForm />);
```

Comment:
the storiesOf is an old form of exporting/creating stories. maybe have a look at this: https://storybook.js.org/docs/formats/component-story-format/ 

however since storybook says it won't be deprecated of course you can still use it. just a small heads up 😉

Reply:
Oh! I just like the storiesOf-style, but of course I'll rather use the recommended format. Thank you for the hint! :D